migrations.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
What does this file represent?

Reply:
Im going to delete this once i debug the problem I'm having.